spec/acceptance/anchor_spec.rb

```diff
@@ -15,11 +15,12 @@ class anchorrefresh {
       }
 
       include anchorrefresh
-    DOC
-    it 'effects proper chaining of resources' do
-      apply_manifest(pp, :catch_failures => true) do |r|
-        expect(r.stdout).to match(%r{Anchor\[final\]: Triggered 'refresh'})
-      end
+    MANIFEST
+  end
+
+  it 'applies manifest' do
```

Comment:
This `it` description is going to be difficult to understand in CI results. There will be no indication of what this test is trying to do. Maybe something a little more descriptive up in the describe string?

Reply:
@RandomNoun7 updated to be a more explicit